core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -0,0 +1,351 @@
+    await contributorDashboardPage.expectOpportunityWithPropertiesToExist(
+      SKILL_DESCRIPTIONS[0], TOPIC_NAMES[0], null, '(0%)');
+
+    // Submit suggestion as user0.
+    await contributorDashboardPage.clickOpportunityActionButton(
+      SKILL_DESCRIPTIONS[0], TOPIC_NAMES[0]);
+    await skillEditorPage.confirmSkillDifficulty();
+    await explorationEditorMainTab.setContent(
+      await forms.toRichText('Question 1'), true);
+    await explorationEditorMainTab.setInteraction('TextInput');
+    await explorationEditorMainTab.addResponse(
+      'TextInput', await forms.toRichText('Correct Answer'), null, false,
+      'FuzzyEquals', ['correct']);
+    await (await explorationEditorMainTab.getResponseEditor(0)).markAsCorrect();
+    await (
+      await explorationEditorMainTab.getResponseEditor('default')
+    ).setFeedback(await forms.toRichText('Try again'));
+    await explorationEditorMainTab.addHint('Hint 1');
+    await explorationEditorMainTab.addSolution('TextInput', {
+      correctAnswer: 'correct',
+      explanation: 'It is correct'
+    });
+    await skillEditorPage.saveQuestion();
+    await users.logout();
+
+    // Review and accept the suggestion as user1.
+    await users.login(USER_EMAILS[1]);
+    await contributorDashboardPage.get();
+    await contributorDashboardPage.waitForOpportunitiesToLoad();
+
+    if (width < 700) {
+      let opportunityItemSelector = function() {
+        return $$('.e2e-test-opportunity-list-item');
+      };
+      let opportunity = (await opportunityItemSelector())[0];
+      await action.click('Opportunity Item', opportunity);
+    } else {
+      await contributorDashboardPage.clickOpportunityActionButton(
+        'Question 1', SKILL_DESCRIPTIONS[0]);
+    }
+    await (
+      contributorDashboardPage.waitForQuestionSuggestionReviewModalToAppear());
+    await contributorDashboardPage.clickAcceptQuestionSuggestionButton();
+    await contributorDashboardPage.waitForOpportunitiesToLoad();
+    await contributorDashboardPage.expectEmptyOpportunityAvailabilityMessage();
+    await users.logout();
+
+    // The below lines enable the cd_admin_dashboard_new_ui flag in dev mode.
+    // They should be removed after the cd_admin_dashboard_new_ui flag is
+    // deprecated.
+    await users.login(ADMIN_EMAIL);
+
+    await releaseCoordinatorPage.getFeaturesTab();
+    var CdAdminDashboardNewUiFlag = (
+      await releaseCoordinatorPage.getCdAdminDashboardNewUiFeatureElement());
+    await releaseCoordinatorPage.enableFeature(
+      CdAdminDashboardNewUiFlag);
+
+    await users.logout();
+  });
+
+  it('should allow user to navigate to new dashboard', async function() {
+    await users.login(QUESTION_COORDINATOR_EMAIL);
+    await contributorDashboardAdminPage.get();
+
+    var questionSubmitterTab = $('.e2e-test-question-submitters-tab');
+    var questionReviewerTab = $('.e2e-test-question-reviewers-tab');
+
+    await action.click('Question Submitter tab', questionSubmitterTab);
```

Comment:
Ditto define these in utils files, follow this pattern for all other palces*

Reply:
Done. PTAL.